packages/markers/src/browser/problem/problem-widget.ts

```diff
@@ -111,16 +120,23 @@ export class ProblemWidget extends TreeWidget {
             case 1: return 'fa fa-times-circle error';
             case 2: return 'fa fa-exclamation-circle warning';
             case 3: return 'fa fa-info-circle information';
-            case 4: return 'fa fa-hand-o-up hint';
-            default: return '';
+            default: return 'fa fa-hand-o-up hint';
         }
     }
 
     protected decorateMarkerFileNode(node: MarkerInfoNode, caption: h.Child): h.Child {
         const fileIcon = this.iconProvider.getFileIconForURI(node.uri);
         const filenameDiv = h.div({ className: fileIcon }, node.uri.displayName);
-        const pathDiv = h.div({ className: 'path' }, node.uri.path.toString());
+        const pathDiv = h.div({ className: 'path' }, this.getRelativePath(node));
         const counterDiv = h.div({ className: 'counter' }, node.numberOfMarkers.toString());
         return h.div({ className: 'markerFileNode' }, filenameDiv, pathDiv, counterDiv);
     }
+
+    protected getRelativePath(node: MarkerInfoNode): string {
+        const absPath = node.uri.parent.toString();
+        if (this.workspacePath) {
+            return absPath.substr(this.workspacePath.length);
+        }
+        return '';
```

Comment:
It should return the full path (i.e. `absPath`) not an empty string in the case of no workspaceRoot.